pootle/apps/pootle_store/models.py

```diff
@@ -1324,7 +1324,7 @@ def real_path(self):
         return self.file.name
 
     @property
-    def is_terminology(self):
+    def is_store_terminology(self):
```

Comment:
Perhaps it is my poor knowledge of English talking, but `is_store_terminology` looks alien to me.